packages/playgrounds/src/sharing.jsx

```diff
@@ -4,7 +4,7 @@ import { Provider } from 'react-redux'
 import ReactDOM from 'react-dom'
 
 import SharingProvider, { ShareButton, ShareModal } from 'cozy-sharing'
-import withLocales from 'cozy-sharing/dist/withLocales'
```

Comment:
From a fresh repo, the withLocales.jsx is not there, I am not sure we need the fix.

```
ls node_modules/cozy-sharing/dist/ | grep withLocale
withLocales.js
```

Reply:
I tried to reproduce the issue but with no success. Certainly a previous debug 🤷‍♂️ . Reverted the commit